### PR TITLE
In the inquiry modal change 'Add Message' to 'Add message'

### PR DESCRIPTION
**Pull request by lilyfromseattle**
[PURCHASE-2276]

Screen Shot 2020-12-03 at 12 05 16 PM

[PURCHASE-2276]: https://artsyproduct.atlassian.net/browse/PURCHASE-2276

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The CHANGELOG is not valid YML:
YAMLSyntaxError: Document is not valid YAML (bad indentation?)
    at /home/circleci/project/node_modules/yaml/dist/Document.js:219:28
    at Array.forEach (<anonymous>)
    at Document.parseContents (/home/circleci/project/node_modules/yaml/dist/Document.js:215:14)
    at Document.parse (/home/circleci/project/node_modules/yaml/dist/Document.js:148:10)
    at parseDocument (/home/circleci/project/node_modules/yaml/dist/index.js:109:37)
    at Object.parse (/home/circleci/project/node_modules/yaml/dist/index.js:120:15)
    at Object.<anonymous> (dangerfile.ts:103:35)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.requireFromString [as default] (/home/circleci/project/node_modules/require-from-string/index.js:28:4)
    at Object.<anonymous> (/home/circleci/project/node_modules/danger/distribution/runner/runners/inline.js:144:63)</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.yml [hunk 1]
```diff
@@ -10,7 +10,8 @@ upcoming:
     - Add artist past shows screen - mounir
     - Add loading spinner to infinite artwork grid on artist page - david
     - Avoid shrinking the background on non-fullscreen fancy modals - david
-releases:
+    - changed casing of Add Message to Add message in inquiry modal - lily
+    releases:
```
Comment (ashfurrow) [inline CHANGELOG.yml:14]: This is the change that Danger is complaining about, we need to un-indent this line and it should work again 👍

**Review by ashleyjelks — APPROVED** [review]

File: CHANGELOG.yml (hunk 1 above)
Comment (ashleyjelks) [reply CHANGELOG.yml:14]: I think this is a trivial enough change to not warrant a Changelog update?

Comment (ashfurrow) [reply CHANGELOG.yml:14]: Yeah, good point! Either way works for me 👍

**Pull request merged**